wazuhapp/README.md

```diff
@@ -25,7 +25,7 @@ To build a Kibana app package, you need to download this repository and use the
     Usage: ./generate_wazuh_app.sh [OPTIONS]
 
         -b, --branch <branch>     [Required] Select Git branch or tag e.g. 3.8-6.7 or v3.7.2-6.5.4
-        -s, --store <path>        [Optional] Set the destination path of package, by defauly /tmp/wazuh-app.
+        -s, --store <path>        [Optional] Set the destination path of package, by defauly splunkapp/output/
```

Comment:
In this case, I would change it to `/path/to/generate_wazuh_app.sh` to remove any hard-coded path that may confuse the user.

Reply:
Done in https://github.com/wazuh/wazuh-packages/pull/305/commits/1dedde988334f5d9ba81861915474ca4e99b6c10